var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -76,9 +76,12 @@ def install(self, spec, prefix):
             'NVCC          = echo',
             'NVCCFLAGS     =',
         ])
-        if '+fpic' in spec:
+        if '+pic' in spec:
             make_args.extend(['CFLAGS=-fPIC', 'FFLAGS=-fPIC'])
```

Comment:
see https://github.com/LLNL/spack/blob/develop/lib/spack/spack/compilers/clang.py#L117-L119, for example